platforms/gke/base/_shared_config/workloads_variables.tf

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Configuration dependencies
+#
+
+locals {
+  manifests_directory = abspath("${path.module}/../manifests")
+}
+
+variable "kueue_version" {
+  default = "0.8.0"
```

Comment:
In the batch ref repo, kueue version is 0.8.1. I think we should harmonise versions